src/generate/renders/generated/CliCommands.ts

```diff
@@ -4,313 +4,215 @@
  *-------------------------------------------------------------------------------------------- */
 
 import { CodeModelAz } from '../../CodeModelAz';
-import { HeaderGenerator } from '../Header';
-import {
-    ToMultiLine,
-    getExtraModeInfo,
-    composeParamString,
-    isNullOrUndefined,
-} from '../../../utils/helper';
+import { isNullOrUndefined } from '../../../utils/helper';
 import { TemplateBase } from '../TemplateBase';
-import { CodeGenConstants, CodeModelTypes, RenderInput, SortOrder } from '../../../utils/models';
-import { SchemaType } from '@azure-tools/codemodel';
-
-let showCommandFunctionName: string;
-let useResourceType: boolean;
-
-function initVars() {
-    showCommandFunctionName = undefined;
-    useResourceType = false;
-}
-
+import {
+    AzConfiguration,
+    CodeGenConstants,
+    CodeModelTypes,
+    PathConstants,
+    RenderInput,
+    SortOrder,
+} from '../../../utils/models';
+import * as path from 'path';
 export class CliCommands extends TemplateBase {
     constructor(model: CodeModelAz) {
         super(model);
+        if (model.IsCliCore) {
+            this.relativePath = path.join(
+                PathConstants.generatedFolder,
+                PathConstants.commandsFile,
+            );
+        } else {
+            this.relativePath = path.join(
+                model.AzextFolder,
+                PathConstants.generatedFolder,
+                PathConstants.commandsFile,
+            );
+        }
+        this.tmplPath = path.join(PathConstants.templateRootFolder, 'generated/commands.py.njx');
     }
 
-    public getDataFromModel() {
-        let data = {};
+    public getDataFromModel(data: any) {
+        data['imports'].push([
+            CodeGenConstants.DEFAULT_CLI_CORE_LIB + '.commands',
+            ['CliCommandType'],
+        ]);
+        let importProfile = false;
+        let lineTooLong = false;
+        let needWaitCommand = false;
+        let showCustomFunctionName = '';
+        const extraNonStringProperties = ['resourceType', 'mode'];
+        const extraProperties = ['maxApi', 'minApi'];
+        const pythonString = (str) => {
+            return "'" + str + "'";
```

Comment:
could you try to use  `'{str}'` for string combination?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
changed.